module/module.py

```diff
@@ -956,12 +957,20 @@ def check_authentication(self, username, password):
                 return False
 
             user = User.from_contact(c)
+            logger.info("[WebUI] User minimum business impact: %s", user.min_business_impact)
```

Comment:
Should be debug.

Reply:
This log only happens on user login. I think it is better to remain at INFO level to have it in the log file.